### ISSUE TITLE
Vagrant box does not compile

### ISSUE THREAD

**Issue by chatelao** [adafruit/Adafruit-PXT-Setup#1]
The box does not fully, start. Adding this on line 32 helped a bit:
```
# Another bug, fix compatibilty
sudo chown -R 1000:1000 "/home/vagrant/.npm"
```

The next command failed too:
```
pxt staticpkg -o /var/www/html
```

Running ```rm -rf /var/www/html``` help a little bit, but still a lot of errors.

PS: Found this repository in the tutorial: https://learn.adafruit.com/installing-makecode-for-adafruit/installing-with-vagrant

PS2: I would like to debug the buggy sound of "Circuit Playground Bluefruit" ... a long way to go :-)

**Comment by ladyada** [comment]
yeah we have not tried it for many years. the person who worked on it is not at adafruit so we dont have any plans to revisit the project. if you get the vagrant working again let us know how and if you submit a PR we could fix it!

**Comment by chatelao** [comment]
Thanks for the fast and honest feedback, like this better than "maybe" or "one day".

Using the microphone in MakeCode crashes the "Circuit Playground Bluefruit" and I tried to get a debugger environment (bought the segger mini edu from you recommendation) . - I will continue a few days to search, the whole CODAL / PXT concepts are very interesting.

**Comment by ladyada** [comment]
well the good news is that we're pretty sure the microbit v2 uses the same kind of microphone and its the same chip
@mmoskal do you have any recommendations for transplanting the microbit v2 mic to the CPB?

**Comment by chatelao** [comment]
Just for traceability the minimal testcase (The micro was tested successfully with CircuitPython):

```javascript
input.onLoudSound(function () {
    pins.LED.digitalWrite(true)
})
```
The original Project on MakeCode: https://makecode.com/_aw47LYPtJ1UF
![image](https://user-images.githubusercontent.com/2461532/107488820-2d9e3780-6b88-11eb-90fa-ec4ee059ac71.png)

And the ".uf2" file in case someone knows how to decompile it:
[maker-adafruit-circuit-playground-bluefruit.uf2.zip](https://github.com/adafruit/Adafruit-PXT-Setup/files/5956995/maker-adafruit-circuit-playground-bluefruit.uf2.zip)
